app/controllers/orders_controller.rb

```diff
@@ -31,16 +31,16 @@ def edit
   def create
     @order = Order.new(order_params)
     @order.add_line_items_from_cart(@cart)
-
+    @order.user = @user if(@user = User.find_by_id(session[:user_id]))
```

Comment:
create a helper_method `current_user` in application_controller which will return the logged_in user
and use here like

`@order.user = current_user`

Let me know if you have any doubts